src/style.css

```
@@ -0,0 +1,81 @@
+/* @font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttf);
+} */
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.container {
+  position: relative;
+  max-width: 100%;
+  margin: 20px 0;
+  border-radius: 4px;
+}
+
+.input {
+  font-family: Avenir, Avenir-Book, Avenir-Heavy, sans-serif;
```

set font-family to the `html` tag selector and let inputs inherit it 
```suggestion
  font-family: inherit;
```